GSA/digital-gov-drupal
# Digital-233: Creating the homepage content

## Jira ticket




[DIGITAL-233](https://cm-jira.usa.gov/browse/DIGITAL-233)

## Purpose



This PR will set up/add a landing page type for home page and migrate over templets along with placeholder for future work

## Includes the following PRs that must be merged first



## Deployment and testing

### Local Setup



`lando rebuild -y`

### QA/Testing instructions



1. go to content and check the page title `Better websites. Better government.` and verify that it has homepage for landing page type and placeholder are present 
2. review twig file and check for overall code quality

## Checklist for the Developer




## Checklist for the Peer Reviewers
